src/components/Speciality/Books.js

```diff
@@ -54,7 +57,7 @@ const BooksSection = ({ books, title }) =>
           <CenteredCol width={[1]}>
             <SectionTitle small>{`${title.trim()} books`}</SectionTitle>
             <BodyPrimary>
-              Node.js books written by members of YLD for our community.
+              {`${title.trim()} books written by members of YLD for our community.`}
```

Comment:
Nice :+1:

I wonder whether we want this line to be editable by Greta? :thinking:

Reply:
The line being hard-coded tells us that we only use Books.js for our Node.js speciality so I would not worry about making it editable at this point in time; but could be it could be the case in the future.